MidasCivil_Adapter/CRUD/Create/Loads/BarVaryingDistributedLoads.cs

```diff
@@ -43,25 +46,21 @@ private bool CreateCollection(IEnumerable<BarVaryingDistributedLoad> barVaryingD
 
             foreach (BarVaryingDistributedLoad barVaryingDistributedLoad in barVaryingDistributedLoads)
             {
-                BarVaryingDistributedLoad load = barVaryingDistributedLoad.ShallowClone();
-                if (!load.RelativePositions)
-                {
-                    Engine.Base.Compute.RecordError("The midas adapter can only handle BarVaryingDistributedLoads with relative positions. Please update the loads to be set with this format.");
-                    continue;
-                }
+              BarVaryingDistributedLoad load = barVaryingDistributedLoad.ShallowClone();
 
-                if (load.StartPosition >= load.EndPosition)
-                {
-                    Engine.Base.Compute.RecordError("Midas civil only supports start positions less than end positions for BarVaryingDistributedLoads.");
-                    continue;
-                }
+              if (load.StartPosition >= load.EndPosition)
+              {
+               Engine.Base.Compute.RecordError("Midas civil only supports start positions less than end positions for BarVaryingDistributedLoads.");
+                continue;
+              }
```

Comment:
In here, before you call the `Convert` method you need to look at the `BarVaryingDistributedLoad` with `RelativePositions = false` and group them based on length. I suggest you append the `BarLength` to the end of the the name. For example, if you have a BarVaryingDistributedLoad (where `Name = BVDL` and `StartPosition = 0.1m` and `EndPosition = 0.2m`) with three bars of 0.5m, 1.0m and 1.5m you should get the three loads back with the names:
1. BDVL-0.500m
2. BDVL-1.000m
3. BDVL-1.500m

Because all of the `StartPosition` and `EndPosition` are going to be different along the bar.